src/tjs.h

```diff
@@ -34,8 +34,9 @@
 
 typedef struct TJSRuntime TJSRuntime;
 
-TJSRuntime *TJS_NewRuntime(void);
+TJSRuntime *TJS_NewRuntime(RunOption *option);
 TJSRuntime *TJS_NewRuntime2(bool is_worker);
+TJSRuntime *TJS_NewRuntimeByOptions(bool is_worker, RunOption *option);
```

Comment:
Let's call this TJS_NewRuntime2 instead and skip the `is_worker` parameter. I'll make the current TJS_NewRuntime2 private. You can rename it to TJS_NewRuntimeWorker in the interim.